sections/errorhandling/centralizedhandling.spanish.md

```diff
@@ -79,8 +175,8 @@ app.use((err, req, res, next) => {
 
 ### Cita de blog: "Los errores de HTTP no deberían existir en el código de tu base de datos"
 
- Del blog Daily JS clasificado 3to para las palabras clave “Node.js error handling”
+ Del blog Daily JS clasificado 14vo para las palabras clave “Node.js error handling”
 
  > ……You should set useful properties in error objects, but use such properties consistently. And, don’t cross the streams: HTTP errors have no place in your database code. Or for browser developers, Ajax errors have a place in the code that talks to the server, but not code that processes Mustache templates…
 
- > ……Deberías setear propiedades útiles en los objetos de error, y usar esas propiedades consistentemente. Y no mezcles los flujos: los errores de HTTP no deberían existir en el código de tu base de datos. O para quienes desarrollan para navegadores, los errores de Ajax pertenecen al código que habla con el servidor, pero no al código que procesa los templates de Mustace…
+ > ……Deberías asignar propiedades útiles en los objetos de error, y usar esas propiedades consistentemente. Y no mezcles los flujos: los errores de HTTP no deberían existir en el código de tu base de datos. O para quienes desarrollan para navegadores, los errores de Ajax pertenecen al código que habla con el servidor, pero no al código que procesa los templates de Mustache…
```

Comment:
"setear" is more regional, "asignar" is more used.
Type on Mustace, should be Mustac**h**e